ui/src/dashboards/utils/getTimezoneOffset.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * PLEASE READ
```

Comment:
suggestion:  ✂️ the 'please read' part

a big comment implies it should be read. 'please read' almost comes off like the comment is begging for attention, like `pls respond`

Reply:
You got it